libraries/functional-tests/slacktestbot/README.md

```diff
@@ -0,0 +1,131 @@
+# Slack functional test pipeline setup
+
+This is a step by step guide to setup the Slack functional test pipeline.
+
+## Slack Application setup
+
+We'll need to create a Slack application to connect with the bot.
+
+1. Create App
+
+    Create a Slack App from [here](https://api.slack.com/apps), associate it to a workspace.
+
+    ![Create Slack App](./media/SlackCreateSlackApp.png)
+
+2. Get the Signing Secret and the Verification Token
+
+    Keep the Signing Secret and the Verification Token from the Basic Information tab.
+
+    These tokens will be needed to configure the pipeline.
+
+    - Signing Secret will become *SlackTestBotSlackClientSigningSecret*.
+    - Verification Token will become *SlackTestBotSlackVerificationToken*.
+
+    ![App Credentials](./media/SlackAppCredentials.png)
+
+3. Grant Scopes
+
+    Go to the OAuth & Permissions tab and scroll to the Scopes section.
+
+    In the Bot Token Scopes, add chat:write, im:history, and im:read using the Add an Oauth Scope button.
+
+    ![Grant Scopes](./media/SlackGrantScopes.png)
+
+4. Install App
+
+    On the same OAuth & Permissions tab, scroll up to the OAuth Tokens & Redirect URLs section and click on Install to Workspace.
+
+    A new window will be prompted, click on Allow.
+
+    ![Install App](./media/SlackInstallApp.png)
+
+5. Get the Bot User OAuth Access Token
+
+    You will be redirected back to OAuth & Permissions tab, keep the Bot User OAuth Access Token.
+
+    - Bot User OAuth Access Token will become *SlackTestBotSlackBotToken* later in the pipeline variables.
+
+    ![OAuthToken](./media/SlackOAuthToken.png)
+
+6. Get the Channel ID
+
+    Go to the Slack workspace you associated the app to. The new App should have appeared; if not, add it using the plus sign that shows up while hovering the mouse over the Apps tab.
+
+    Right click on it and then on Copy link.
+
+    ![ChannelID](./media/SlackChannelID.png)
+
+    The link will look something like *https://workspace.slack.com/archives/**N074R34L1D***.
```

Comment:
The intent with this was to get the code in bold? it's looking like this
![image](https://user-images.githubusercontent.com/38112957/100109229-63196b80-2e4a-11eb-9d88-b2b5644a2e8a.png)

Reply:
It was, I corrected it just now.